.travis.yml

```diff
@@ -9,7 +9,7 @@ before_install:
   - bundle --version
 bundler_args: --without=development
 
-rvm: 2.0.0
+rvm: 2.2.2
```

Comment:
Older Vagrant versions bundle Ruby 2.0 so we really want to test with that version as long as we want to support those Vagrant versions. And I don't think we want to cut the support unless needed.

If the build or tests are broken with the old ruby, we should fix that first. I can take a look later.